src/registrar/templates/admin/change_list_results.html

```diff
@@ -82,7 +74,37 @@
             {{ item }}
         {% endfor %}
     </tr>
-{% endfor %}
+    {% endfor %}
+
+{% else %}
```

Comment:
*suggestion*: This else is pretty far away from it's if. Could we use a short `{# ... #}` comment to explain what case this corresponds to?
```suggestion
{% else %} {# results doesn't have a form as its first element #}
```

Reply:
Will do